test/test_bbox.py

```diff
@@ -219,20 +219,11 @@ def test_bbox_path_stroke_unset(self):
         ))
 
     def test_bbox_subpath(self):
```

Comment:
This test gets gutted for no reason. It's still the case that translating by 2 should set the changed bounds.

Reply:
```python

    def test_bbox_move_subpath(self):
        p = Path("M 0,0 Z m 100,100 h 20 v 20 h -20 Z")
        e = p.subpath(1)
        self.assertEqual(e.bbox(), (100, 100, 120, 120))        
```

Might make a better test since it's really clear that the bounding box should be what is given.

Reply:
Why do you say that this test has been gutted? It still has a a translation?

The sizes / positions /  translation values were chosen to be unque.

Is there a particular reason that a translation of 2 is required to test this (rather than 5)?